docs/admin/README.md

```diff
@@ -49,10 +49,10 @@ uses OpenVSwitch to set up networking between pods across
   Kubernetes nodes.
 
 If you are modifying an existing guide which uses Salt, this document explains [how Salt is used in the Kubernetes
-project.](salt.md).
+project](salt.md).
 
-## Upgrading a cluster
-[Upgrading a cluster](cluster-management.md).
+## Managing a cluster
```

Comment:
Thanks for noticing this. To make sure this gets in before the deadline, I created a separate PR with this change (#11560)